.github/workflows/validate-pr.yml

```diff
@@ -0,0 +1,21 @@
+name: "Validate PRs"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
```

Comment:
Assuming `synchronize` means merging with master (I can't seem to find what this is), is this part needed?

Reply:
https://github.com/orgs/community/discussions/24567